### PR TITLE
cb248 Minors 9

### PR DESCRIPTION
**Pull request by Brandonz07**
Point 6 "footer" was resolved in task cb245 "minor6"

### CODE REVIEW

File: src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx [hunk 1]
```diff
@@ -386,8 +386,11 @@ export const ComercialManagement = (props: ComercialManagementProps) => {
           <ReportCreditsModal
```
Comment (SelsaErojas) [inline src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx:386]: Adjust the import aliases using the alias for components and adjust order of the import.
relocate the props accordingly to the standard

File: src/pages/board/outlets/creditProfileInfo/CreditBehaviorCard/index.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { Stack } from "@inubekit/stack";
 
 import { CardInfoContainer } from "@components/cards/CardInfoContainer";
 import { StyledDivider } from "@components/cards/SummaryCard/styles";
-import { formatDateWithFullYear } from "@utils/formatData/date";
+import { formatPrimaryDate } from "@utils/formatData/date";
 import { ItemNotFound } from "@components/layout/ItemNotFound";
 import userNotFound from "@assets/images/ItemNotFound.png";
```
Comment (SelsaErojas) [inline src/pages/board/outlets/creditProfileInfo/CreditBehaviorCard/index.tsx:10]: relocate the props accordingly to the standard (Line 16)

File: src/components/modals/ReportCreditsModal/index.tsx [hunk 3]
```diff
@@ -17,26 +17,37 @@ import {
...
-import { ActionModal } from "./Actions";
-import { Details } from "./Detail";
+import { Detail } from "@components/data/TableExtraordinaryInstallment/Detail";
+import { NewPrice } from "./components/newPrice";
 import { StyledContainerClose, StyledContainer } from "./styles";
 
 export interface ReportCreditsModalProps {
   handleClose: () => void;
+  onChange: (name: string, newValue: string) => void;
   portalId?: string;
```
Comment (SelsaErojas) [inline src/components/modals/ReportCreditsModal/index.tsx:34]: relocate the props accordingly to the standard

File: src/components/modals/ReportCreditsModal/index.tsx [hunk 4]
```diff
@@ -17,26 +17,37 @@ import {
...
 import { Blanket } from "@inubekit/blanket";
 import { useMediaQuery } from "@inubekit/hooks";
 import { SkeletonLine, SkeletonIcon } from "@inubekit/skeleton";
+import { Select } from "@inubekit/select";
 
 import { usePagination } from "./utils";
 import { headers, data, dataReport } from "./config";
-import { ActionModal } from "./Actions";
-import { Details } from "./Detail";
+import { Detail } from "@components/data/TableExtraordinaryInstallment/Detail";
```
Comment (SelsaErojas) [inline src/components/modals/ReportCreditsModal/index.tsx:27]: adjust and relocate order of the import

File: src/components/inputs/SummaryOnProspect/styles.ts [hunk 5]
```diff
@@ -26,6 +26,7 @@ export const Container = styled.div<IStyledCollapseIcon>`
   display: flex;
   gap: ${({ $showIcon }) => ($showIcon ? "0px" : "64px")};
   padding: 4px;
+  width: ${({ $showIcon }) => ($showIcon ? "auto" : "100%")};
 `;
 
 export const IconWrapper = styled.div`
```
Comment (SelsaErojas) [inline src/components/inputs/SummaryOnProspect/styles.ts:32]: Adjust to the inubekit library

File: src/components/data/TableExtraordinaryInstallment/index.tsx [hunk 6]
```diff
@@ -61,7 +61,7 @@ const usePagination = (data: IExtraordinaryPayment[]) => {
 export const TableExtraordinaryInstallment = (
```
Comment (SelsaErojas) [inline src/components/data/TableExtraordinaryInstallment/index.tsx:61]: Adjust the import aliases using the alias for components

File: src/components/data/TableExtraordinaryInstallment/config/index.tsx [hunk 7]
```diff
@@ -12,9 +12,9 @@ export const rowsActions = [
     {
```
Comment (SelsaErojas) [inline src/components/data/TableExtraordinaryInstallment/config/index.tsx:12]: Adjust the import aliases using the alias for components

**Comment by Brandonz07** [comment]
The variable names are updated when the main merge is done

**Review by SelsaErojas — APPROVED** [review]
Functional

**Review by carin91 — APPROVED** [review]

**Pull request merged**